examples/log_sig_example.py

```diff
@@ -1,13 +1,14 @@
 # Example: creating a LogicSig transaction signed by a program that never approves the transfer.
 
 import tokens
-from algosdk import algod, account
+from algosdk import account
+from algosdk.v2client import algod
```

Comment:
@algochoi Minor:  Unless opposed, can you update the PR to be sandbox-compatible?

I think you raised a good point in our offline chat.  I suspect others are more likely to test via sandbox as a default.  

It feels to me like a minimally involved improvement to _examples_.  With the understanding that a more in-depth treatment is out-of-scope.

Reply:
That sounds like a good improvement - I tried addressing this in [aafcf35](https://github.com/algorand/py-algorand-sdk/pull/403/commits/aafcf352914e2b210ec3aeedfe44f85da4007ca2)